workflow-service-sdk/pom.xml

```diff
@@ -0,0 +1,203 @@
+                                <fileset>
+                                    <directory>${basedir}</directory>
+                                    <includes>
+                                        <include>**</include>
+                                    </includes>
+                                    <excludes>
+                                        <exclude>.openapi-generator-ignore</exclude>
+                                        <exclude>pom.xml</exclude>
+                                        <exclude>README.md</exclude>
+                                    </excludes>
+                                    <followSymlinks>false</followSymlinks>
+                                </fileset>
+                            </filesets>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>com.googlecode.maven-download-plugin</groupId>
+                <artifactId>download-maven-plugin</artifactId>
+                <version>1.6.8</version>
+                <executions>
+                    <execution>
+                        <phase>initialize</phase>
+                        <goals>
+                            <goal>wget</goal>
+                        </goals>
+                        <configuration>
+                            <url>http://localhost:8080/v3/api-docs</url>
+                            <outputFileName>api-docs.json</outputFileName>
+                            <outputDirectory>${basedir}</outputDirectory>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.openapitools</groupId>
+                <artifactId>openapi-generator-maven-plugin</artifactId>
+                <version>${openapi.generator.version}</version>
+                <executions>
+                    <execution>
+                        <id>generate-client-api-code</id>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <generatorName>java</generatorName>
+
+                            <groupId>dev.parodos</groupId>
+                            <artifactId>workflow-service-sdk</artifactId>
+                            <packageName>com.redhat.parodos.openapi</packageName>
+
+                            <inputSpec>${project.basedir}/api-docs.json</inputSpec>
+                            <output>${basedir}</output>
+
+                            <invokerPackage>com.redhat.parodos.openapi.api</invokerPackage>
+                            <apiPackage>com.redhat.parodos.openapi.api</apiPackage>
+                            <modelPackage>com.redhat.parodos.openapi.model</modelPackage>
+                            <modelNameSuffix>Generated</modelNameSuffix>
```

Comment:
I don't think this is needed. The name of the package should highlight its origin.
We can change the modelPackage to `com.redhat.parodos.sdk.model` so it will be clear this isn't the model.